app/models/user.rb

```diff
@@ -1,6 +1,37 @@
 class User < ActiveRecord::Base
+  validates :beard_length, :numericality => { :greater_than_or_equal_to => 0 }
 
   def fullname
     first_name + " " + last_name
   end
+
+  def validate_trim_length(length)
```

Comment:
Seems this method is not part of User's public interface. Maybe it is better to do this method private?